.github/workflows/main.yml

```diff
@@ -251,6 +251,7 @@ jobs:
       - name: Building Playground application
         run: yarn playground:build
         env:
+          NODE_OPTIONS: '--max-old-space-size=6144'
```

Comment:
Ok so it seems to work but we need to bump the memory.

Maybe we make it an opt-in option to generate the sourcemaps.

Reply:
I think this is a great idea 👍